spark/build.sbt

```diff
@@ -1,14 +1,21 @@
 name := "smile-spark"
 
+scalaVersion := "2.12.8"
+
 // Parent project disables Scala as most libraries are in Java.
 // Enable it as this is a Scala project.
 crossPaths := true
 autoScalaLibrary := true
 
+fork.in(Test) := true
+parallelExecution.in(Test) := false
+
 libraryDependencies ++= {
   val sparkV = "2.4.7"
   Seq(
     "org.apache.spark" %% "spark-core" % sparkV % Provided,
-    "org.apache.spark" %% "spark-sql"  % sparkV % Provided
+    "org.apache.spark" %% "spark-sql"  % sparkV % Provided,
+    "org.apache.spark" %% "spark-mllib"  % sparkV % Provided,
```

Comment:
What SparkSQL features are needed?

Reply:
The project was already depending on spark-sql and we need it for Row, DataFrame, etc. And Spark MLLib depends on SparkSQL anyway